### DISCUSSION TITLE
Welcome to erebus Discussions!

### DISCUSSION THREAD

**Discussion by ethicalhackingplayground** [ethicalhackingplayground/erebus#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shizzgar** [comment]
Чувак, ты просто лучший 🤟
Это то что я хотел получить в идеале от nuclei!
Буду активно использовать и наблюдать за дальнейшем развитием проекта 🤓
Не бросай проект, это только начало🔝

------------------------------------------------------
Yo, dude, your work is awesome!
This is definitely what I, when a was using nuclei.
I will use from time to time with my work and  watch  the development of Your project.
This is just beginning!
